xm-commons-lep/src/main/java/com/icthh/xm/commons/lep/XmLepScriptConfigServerResourceLoader.java

```diff
@@ -31,6 +31,12 @@
     // /config/tenant/{tenant-key}/{ms-name}/lep/**
     private final String tenantLepScriptsAntPathPattern;
 
+    // /config/tenant/{tenant-key}/commons/lep/**
+    private static final String commonsLepScriptsAntPathPattern = "/config/tenants/{tenantKey}/commons/lep/**";
+
+    // /config/tenant/commons/lep/**
+    private static final String environmentLepScriptsAntPathPattern = "/config/tenants/commons/lep/**";
```

Comment:
Seems from now it is not allowed now to create tenant with the name `commons`.

Suggest checking the following places where pattern is used:
1. `com.icthh.xm.commons.tenant.TenantContextUtils.TENANT_KEY_REGEXP`
2. `resources/swagger-spec/tenant/swagger-tenant-spec.yaml`  line 145
3. `com.icthh.xm.commons.tenant.TenantKeyUnitTest#testValidateTenantKey`

Reply:
Done:
- refactor TenantScriptStorage according to https://github.com/xm-online/xm-commons/tree/feature-add-support-commons-for-all-apps-refactor
- exclude commons tenant name from TENANT_KEY_REGEXP
- add unit tests on tenant name exclusion